encryptor/utils_test.go

```diff
@@ -103,6 +107,64 @@ inner join table6 on table6.col1=t1.col1
 			}
 		}
 	})
+	t.Run("with aliased table and non-aliased colum name", func(t *testing.T) {
+		testcases := []struct {
+			query   string
+			dialect dialect.Dialect
+		}{
+			{
+				query:   `SELECT "id", "email", "mobile_number" AS "mobileNumber" FROM "users" AS "User" where "User"."is_active"`,
```

Comment:
lets use a bit more complicated queries with several aliased tables in the `FROM` statement to be sure that we use first from the list